.github/workflows/transfer_file.yml

```diff
@@ -0,0 +1,33 @@
+name: Transfer file
+
+# Controls when the workflow will run
+on:
+  workflow_dispatch:
+    inputs:
+      FILENAME:
+        type: string
+        default: path/filename.json
```

Comment:
Can you change this to required and remove the default? 

A default does not make sense for this use case